test/integration/toolchains/cmake/cmakedeps/test_cmakedeps.py

```diff
@@ -726,6 +726,32 @@ def package_info(self):
     for variable in cmake_variables:
         assert f"CMakeFileName_{variable}" in dep_config
 
+    consumer_conanfile = textwrap.dedent("""
+        from conan import ConanFile
+        from conan.tools.cmake import CMakeDeps
+        class Pkg(ConanFile):
+            name = "pkg"
+            version = "0.1"
+            settings = "os", "build_type", "arch", "compiler"
+
+            def requirements(self):
+                self.requires("dep/1.0")
+
+            def generate(self):
+                deps = CMakeDeps(self)
+                deps.set_property("dep", "cmake_additional_variables_prefixes", ["PREFIX", "prefix", "PREFIX"])
```

Comment:
Repeated PREFIX; this is on purpose?

Reply:
Yes, the tests check that `assert dep_config.count("PREFIX_VERSION") == 1` later for another case, this is just copy/paste of that